Cython/Compiler/PyrexTypes.py

```diff
@@ -153,6 +153,10 @@ def check_for_null_code(self, cname):
         """
         return None
 
+    def get_container_type(self) -> str:
+        """Equivalent to typing.get_origin()"""
+        return None
```

Comment:
I wouldn't want to look up the documentation of `typing.get_origin()` just to understand this docstring.
And why return a type name instead of the type?
```suggestion
    def get_container_type(self) -> Optional[BuiltinObjectType]:
        """Returns the basic container type of the (potentially subscripted) type,
        or None if not a container type.

        Similar to typing.get_origin().
        """
        return None
```